app/src/main/res/values/strings.xml

```diff
@@ -56,70 +56,70 @@
-    <string name="text_application">Application</string>
+    <string name="text_application">App</string>
     <string name="text_image">Image</string>
     <string name="text_music">Music</string>
     <string name="text_video">Video</string>
     <string name="text_showSystemApps">Show system apps</string>
-    <string name="text_showSystemAppsSummary">Show system apps in share screen</string>
+    <string name="text_showSystemAppsSummary">Show system apps in \"Share\" section</string>
     <string name="butn_showSystemApps">@string/text_showSystemApps</string>
     <string name="mesg_notAllowed">You are not allowed</string>
     <string name="text_connectionProblem">Connection problem</string>
     <string name="text_communicationProblem">Communication problem</string>
     <string name="butn_multiSelect">Multi selection</string>
     <string name="butn_receive">Receive</string>
     <string name="butn_block">Block</string>
-    <string name="text_aboutSummary">This software is developed by Genonbeta Open-Source Software Organization</string>
+    <string name="text_aboutSummary">This software is developed by the Genonbeta Libre Software Org</string>
     <string name="butn_appLaunch">Launch</string>
-    <string name="ques_launchApplication">Do you want to launch this application?</string>
-    <string name="mesg_launchApplicationError">This application cannot be launched</string>
+    <string name="ques_launchApplication">Do you want to launch this app?</string>
+    <string name="mesg_launchApplicationError">This app cannot be launched</string>
     <string name="text_enterText">Enter text here…</string>
     <string name="mesg_textCopiedToClipboard">Text copied to clipboard</string>
     <string name="ques_copyToClipboard">Copy to clipboard?</string>
     <string name="text_textReceived">Text received</string>
     <string name="text_receivedTextSummary">Received clipboard text</string>
     <string name="butn_checkForUpdates">Check for updates</string>
     <string name="butn_seeSourceCode">See source code</string>
-    <string name="text_typeText">Type the text you want to send…</string>
+    <string name="text_typeText">Write what you want to send…</string>
     <string name="text_textEditor">Text editor</string>
     <string name="butn_done">Done</string>
     <string name="text_empty">(empty)</string>
     <string name="text_textStream">Text stream</string>
     <string name="butn_wifiSettings">Wi-Fi</string>
     <string name="text_pendingTransfers">Pending transfers</string>
     <string name="text_stopping">Stopping!</string>
-    <string name="text_cancellingTransfer">The job is being cancelled</string>
+    <string name="text_cancellingTransfer">Canceling job…</string>
     <string name="butn_killNow">Kill now</string>
-    <string name="mesg_groupOngoingNotice">This file will be received after %s</string>
+    <string name="mesg_groupOngoingNotice">File complete in %s</string>
```

Comment:
%s is converted to the name of the file which is being received. So I think we should leave as it is.